4iprocessing/generate_initial_config.py

```diff
@@ -0,0 +1,144 @@
+
+import os
+from pathlib import Path
+import re
+from aicsimageio import AICSImage
+import argparse
+import json
+
+# run this to make the initial yaml files and then edit those files to make sure nothing is missing.
+# this code only helps make the yaml files...it does not generate a perfect yaml automatically. 
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--input_dirs', type=str, default=["/allen/aics/microscopy/Antoine/Analyse EMT/4i Data/5500000733","/allen/aics/microscopy/Antoine/Analyse EMT/4i Data/5500000724", "/allen/aics/microscopy/Antoine/Analyse EMT/4i Data/5500000728", "/allen/aics/microscopy/Antoine/Analyse EMT/4i Data/5500000726", "/allen/aics/microscopy/Antoine/Analyse EMT/4i Data/5500000725"], help="input dirs to parse")
```

Comment:
might want to not have these hardcoded as these will change once the isilon is gone?

Reply:
nice catch, took care of this